models/apple_search_ads__ad_group_report.sql

```diff
@@ -1,15 +1,20 @@
+-- See below for requested styling updates
```

Comment:
Can you use this modified model as a template for the styling we should apply. If you could apply the same styling to the other models to be consistent with our other packages that would be great!

The only real styling differences are adding spaces between CTEs to make it easier to read, and entering in the join statements.